include/mbedtls/ecdsa.h

```diff
@@ -236,7 +237,8 @@ int mbedtls_ecdsa_write_signature_det( mbedtls_ecdsa_context *ctx,
 #endif /* MBEDTLS_ECDSA_DETERMINISTIC */
 
 /**
- * \brief           Convert a signature from numbers to ASN.1
+ * \brief           Convert a signature from numbers to ASN.1 INTEGER's,
+ *                  then both packed together as parts of an ASN.1 SEQUENCE
```

Comment:
I think that level of detail is confusing, I actually preferred the previous description. As user shouldn't need to know the details of what the standard ASN.1 representation of an ECDSA signature is.

Reply:
I think it is beneficial to provide the information, but perhaps a `\note` would be more suitable, using the notation
```
Ecdsa-Sig-Value ::= SEQUENCE {
            r       INTEGER,
            s       INTEGER
        }
```
instead of spelling it out.

Reply:
I'm OK with having it in a note if you want, but it doesn't belong in the `'brief` description.

Reply:
Done.